### PR TITLE
CHE-2179: add description widget

### PR DESCRIPTION
**Pull request by ashumilova**
### What does this PR do?

Adds description widget to workspace list, administation page and dashboard page.
### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/2179

Signed-off-by: Ann Shumilova ashumilova@codenvy.com

### CODE REVIEW

**Comment by slemeur** [comment]
+1

File: dashboard/src/components/widget/description/che-description.styl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+$description-text-color = #6e6e6e
+$description-border-color = #cbcbcb
+$description-background-color = #EDEDED
```
Comment (benoitf) [inline dashboard/src/components/widget/description/che-description.styl:3]: should colors bring to the top level .styl ?

Comment (ashumilova) [reply dashboard/src/components/widget/description/che-description.styl:3]: those are used only here, so decided not to define them in styl.

Comment (benoitf) [reply dashboard/src/components/widget/description/che-description.styl:3]: ok seems #CBCBCB color is also used there https://github.com/eclipse/che/blob/a8dfb6bf541940c636590119ef2fb45c1fa659c9/dashboard/src/components/widget/empty-state/che-empty-state.styl#L2

Comment (ashumilova) [reply dashboard/src/components/widget/description/che-description.styl:3]: Fixed, added very-light-grey-color
name was taken from here http://www.htmlcsscolor.com/hex/CBCBCB

**Pull request merged**